Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 1,
-        "Patch": 6
+        "Patch": 7
     },
     "demands": [],
     "minimumAgentVersion": "2.0.0",
```

Comment:
As mentioned below, I will wait for resolution on module.json before rremoving strings from here. Will take it separately.